### ISSUE TITLE
Default pass for tsec user

### ISSUE THREAD

**Issue by linux95** [dtag-dev-sec/tpotce#509]
I installed this application onto an AWS instance. I was able to assign a user and password to the web user. However, once the server rebooted, I was no longer able to ssh into the instance. I was not given the opportunity to assign the tsec user a password. Could you provide the default password for the tsec user so that I can access the instance to finalize the configuration? Thank you.

Dustin

# Issues

Before you post your issue make sure it has not been answered yet and provide `basic support information` if you come to the conclusion it is a new issue:
- 🔍 Use the [search function](https://github.com/dtag-dev-sec/tpotce/issues?utf8=%E2%9C%93&q=) first
- 🧐 Check our [WIKI](https://github.com/dtag-dev-sec/tpotce/wiki)
- 📚 Consult the documentation of 💻 [Debian](https://www.debian.org/doc/), 🐳 [Docker](https://docs.docker.com/), the 🦌 [ELK stack](https://www.elastic.co/guide/index.html) and the 🍯 [T-Pot Readme](https://github.com/dtag-dev-sec/tpotce/blob/master/README.md).
- **⚠️ Provide [basic support information](#info) or similiar information with regard to your issue or we can not help you and will close the issue without further notice**

<br>
<br>
<br>

<a name="info"></a>
## ⚠️ Basic support information (commands are expected to run as `root`)

- What version of the OS are you currently using `lsb_release -a` and `uname -a`?
- What T-Pot version are you currently using?
- What edition (Standard, Nextgen, etc.) of T-Pot are you running?
- What architecture are you running on (i.e. hardware, cloud, VM, etc.)?
- Did you have any problems during the install? If yes, please attach `/install.log` `/install.err`.
- How long has your installation been running?
- Did you install upgrades, packages or use the update script?
- Did you modify any scripts or configs? If yes, please attach the changes.
- Please provide a screenshot of `glances` and `htop`.
- How much free disk space is available (`df -h`)?
- What is the current container status (`dps.sh`)?
- What is the status of the T-Pot service (`systemctl status tpot`)?
- What ports are being occupied? Stop T-Pot `systemctl stop tpot` and run `netstat -tulpen`
- If a single container shows as `DOWN` you can run `docker logs <container-name>` for the latest log entries

**Comment by realitychx** [comment]
I just did a new install this morning, using 19.03, and I was not allowed to move forward without entering a passcode for the tsec user.

That said, I did stumble upon this older reference to a default passcode in the older version of T-Pot

https://dtag-dev-sec.github.io/mediator/feature/2015/03/17/concept.html

user:tsec
pass:tsec

**Comment by t3chn0m4g3** [comment]
If you are installing on AWS you probably did not use the ISO image, hence there is no tsec user.

**Comment by linux95** [comment]
Marco,
Thanks for the response. You are correct. I did not use the ISO image. I used the community Debian image provided by AWS. However, I was able to unmount the drive from the Tpot image and remount to another server. I was able to update the user accounts and ssh certs. Once I did that, I unmounted and re-mounted to the Tpot server and was able to ssh into the server using an admin account. Thanks again.

Dustin

On Jan 5, 2020, at 1:23 PM, Marco Ochse <notifications@github.com<mailto:notifications@github.com>> wrote:


If you are installing on AWS you probably did not use the ISO image, hence there is no tsec user.
